pkg/cli/service.go

```diff
@@ -926,14 +1041,14 @@ func (img imageStruct) string(format string, maxImgNameLen, maxTagLen int) (stri
 	}
 }
 
-func (img imageStruct) stringPlainText(maxImgNameLen, maxTagLen int) (string, error) {
+func (img imageStruct) stringPlainText(maxImgNameLen, maxTagLen, maxPlatformLen int) (string, error) {
```

Comment:
A note here. This should ideally work to some degree with servers which don't enable the graphql api. Mainly to list all images and filter them by name - use just the manifest and catalog calls.